.github/workflows/terraform-azdev-agent-release.yml

```diff
@@ -1,69 +1,47 @@
-name: Release azure devops agent
+name: Scan/Release azure devops agent
```

Comment:
```suggestion
name: Scan/Release azure devops agent Image
```

Reply:
I got confused as I was sure I had seen this before.
Might be better to rename this to be more explicit: https://github.com/Altinn/altinn-platform/pull/1377/files#diff-edc19e8f936ae1a5e55073fbbb3092b18650ae90f12a7ffb485fae1ba857eeb6R1
I ended up going to the READMEs to understand what each image was supposed to be used for.

Reply:
Yeah, had a dejavu feeling myself. I can create a issue about renaming the "base" image for azure devops (and probably the github one aswell) to better reflect that they are ment as small base images to extend if you need some more tools installed

Reply:
https://github.com/Altinn/altinn-platform/issues/1382